### ISSUE TITLE
0.4.1 release issue - could not find matching version

### ISSUE THREAD

**Issue by julian-ganguli** [calogica/dbt-date#49]
I'm trying to upgrade dbt-date after upgrading dbt to 0.21.0. 

Running `dbt deps` returns the following error:

```Running with dbt=0.21.0
Encountered an error:
Could not find a matching version for package calogica/dbt_date
  Requested range: =0.4.1, =0.4.1
  Available versions: ['0.1.1', '0.1.2', '0.1.3', '0.1.4', '0.2.0', '0.2.1', '0.2.2', '0.2.3', '0.2.4', '0.2.5', '0.2.6', '0.3.0', '0.3.1', '0.4.0']
```

packages.yml:
```packages:
  - package: calogica/dbt_date
    version: 0.4.1
```

Github shows `0.4.1` as the latest release but dbt hub still shows `0.4.0`

**Comment by julian-ganguli** [comment]
I was able to get this working by changing the packages.yml (below) and running `dbt deps` afterwards.

```
packages:
  - git: 'https://github.com/calogica/dbt-date.git'
    revision: 0.4.1
```

**Comment by clausherther** [comment]
Hi @julian-ganguli ! Yes, unfortunately dbt hub is a little slow to show the latest versions. I'll check with the dbt labs folks on that. 
FYI, if you use the 
```yaml
 - package: calogica/dbt_date
```
syntax, you're always using dbt hub, which usually is the preferred way. You can also get around the version check by adding `--no-version-check` to your dbt run. 

Thanks!

**Comment by clausherther** [comment]
Closed via #47
